packages/messenger-internal/src/v1/types.ts

```diff
@@ -10,17 +10,32 @@ export interface ApiState {
   getState(): State;
 }
 
+export interface Credentials {
+  token: string;
+  uuid: string;
+}
+
+export interface MountOptions {
+  credentials?: Credentials;
+  openConversationId?: number;
```

Comment:
If we are able to `api.mount().then(() => api.__unstableOpenConversation(id))`, then let's remove this param from options.